src/sage/groups/abelian_gps/abelian_group.py

```diff
@@ -1627,7 +1631,7 @@ def __init__(self, ambient, gens, names="f", category=None):
             Multiplicative Abelian subgroup isomorphic to Z x Z generated by {a^3, b}
             sage: F.subgroup([c])
             Multiplicative Abelian subgroup isomorphic to C2 x C3 x C5 generated by {c}
-            sage: F.subgroup([a, c])
+            sage: S = F.subgroup([a, c]); S
```

Comment:
I don't see why you need to hold onto this subgroup. I would revert this change.